### PR TITLE
Fixed buttons on Android not showing Ripple effect

### LINKED ISSUE #655: No visual changes available on focus state/pressed for some buttons on Android
Scenario to reproduce

Samsung Galaxy J3 Prime
Android version 7.0
App version: 1.0 (46)

For the buttons such as “Enter number”, “Next” there is no focus state indicator when touched. Also, when pressed to activate them no visual change for users to know that they pressed the buttons.

Expected result
Users should be able to see the buttons have focus and they are pressed from the visual change.

WCAG
https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html

Screenshots
button

### PR DESCRIPTION
**Pull request by nickmly**
Fixes #655 

- Switched from custom Ripple solution to use `RectButton` component from `react-native-gesture-handler` for performance and accessibility.

### CODE REVIEW

File: src/components/ButtonMultiline.tsx [hunk 1]
```diff
@@ -59,13 +59,18 @@ export const ButtonMultiline = ({
       }
     : {};
   const externalArrowIcon = textColor === palette.white ? 'icon-external-arrow-light' : 'icon-external-arrow';
-
+  const borderRadius = 4;
```
Comment (amazingphilippe) [inline src/components/ButtonMultiline.tsx:62]: Those single and multi-line buttons are treated like `InfoButton` with a border radius of 10

File: src/components/Button.tsx [hunk 2]
```diff
@@ -69,14 +69,15 @@ export const Button = ({
     : {};
   const externalArrowIcon = textColor === palette.white ? 'icon-external-arrow-light' : 'icon-external-arrow';
 
+  const borderRadius = 4;
```
[the remark this answers is not in the document]
Comment (nickmly) [reply src/components/Button.tsx]: Fixed, thank you!

**Comment by timarney** [comment]
@amazingphilippe - any comments before this is merged?

**Comment by timarney** [comment]
I tested it works well :)

**Comment by timarney** [comment]
Will merge and we can adjust in another PR as need be.

**Pull request merged**